docs/predicate/slsa/v2.md

```diff
@@ -0,0 +1,621 @@
+<!--
+---
+linkTitle: "Chains:v2"
+weight: 10
+---
+-->
+
+# Build Type
+
+This buildType is maintained by [TektonCD/Chains](https://github.com/tektoncd/chains) and describes a build executed by [TektonCD/Pipelines](https://github.com/tektoncd/pipeline).
+
+
+# Description
+
+```
+"buildType": "https://tekton.dev/predicate/chains/slsa/v2"
+```
+
+This buildType describes the execution of a build via a `PipelineRun` or a `TaskRun`.
+
+# Build Definition
+
+## External Parameters
+
+All external parameters are REQUIRED unless otherwise noted. These are fields set by the users in their `PipelineRun/TaskRun` definition.
+
+
+Parameter | Type | Description
+-----------|------|------------
+runSpec | object | complete [pipelineRunSpec](https://tekton.dev/docs/pipelines/pipelineruns/)/[taskRunSpec](https://tekton.dev/docs/pipelines/taskruns/). 
+[buildConfigSource](#build_config_source) | object | The resolved reference to the external build config source.
+
+### Build Config Source
```

Comment:
Perhaps worth adding some context here that build config source in tekton can come from a variety of sources i.e. a git repo, an OCI registry. 

This will make the content of the table self explanatory.